pkg/pgmodel/testdata/expected/info_view-postgres.out

```diff
@@ -37,12 +37,12 @@ SELECT id , metric_name , table_name, retention_period, chunk_interval > interva
 (2 rows)
 
 SELECT * FROM prom_info.label ORDER BY key;
-    key    | value_column_name | id_column_name |        values         
------------+-------------------+----------------+-----------------------
- __name__  | __name__          | __name___id    | {cpu_total,cpu_usage}
- namespace | namespace         | namespace_id   | {dev,production}
- new_tag   | new_tag           | new_tag_id     | {foo}
- new_tag_2 | new_tag_2         | new_tag_2_id   | {bar}
- node      | node              | node_id        | {brain,pinky}
+    key    | value_column_name | id_column_name |        values         | cardinality 
```

Comment:
the column name should be num_values. not sure why it's still cardinality? Did you not update the golden file?

Reply:
the postgres view got left out, and I guess this wasn't caught since these tests are skipped if running on vanilla postgres